share/doc/methods.rst

```diff
@@ -655,17 +657,119 @@ unregularized version of IMC is applied.
     </inverse>
   </non-bonded>
 
+Internal degrees of freedom
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For internal degrees of freedom one can apply the IBI method. In that case
+one also has to provide a setting_nonbonded file, which will be used to 
+calculate the IMC matrix only from the nonbonded interactions.
+
+.. code:: xml
+
+  <inverse>
+    <imc>
+      <bonded_method>ibi</bonded_method>
```

Comment:
I thought about his case before, maybe we should implement ibi as a post update method instead, see 
 e.g. `postupd_cibi_correction.sh`, and then force the user to specific `do_potential` 0 for the bonded interactions.

This approach would have the added benefit that one could then combine IBI with an other updating scheme, not only IMC. And we wouldn't need to reimplement the whole thing for e.g. RE a second time.

Reply:
I think I agree. We would then also change it for the case where IBI is the nonbonded method, right?
I would suggest to make this a separate issue though.

Reply:
Would this lead to any problems with other post update methods like smoothing?

Reply:
I guess, IBI would need to come first and one could i

> I think I agree. We would then also change it for the case where IBI is the nonbonded method, right?
> I would suggest to make this a separate issue though.

Yeah, it would  be cleaner to do it another issue, I mainly want to save you from writing doc that later will be deleted.

> Would this lead to any problems with other post update methods like smoothing?

I guess, IBI would need to come first and one could check for that. On the other hand maybe a user want to do something fancy like `0.5 cibi + 0.5 ibi`. User error can happen in a lot of places.

Reply:
The other reason was that we would have all these changes related to `bonded_method` that we undo later, but by the end of the day it is your call.

Reply:
Then I say, we do this separatedly #676